File: go/Potatoes/Potatoes.go
package main

import "fmt"

func Potatoes(p0, w0, p1 int) int {
	return w0 * (100 - p0) / (100 - p1)
}

func main() {
	fmt.Println(Potatoes(99, 100, 98))
	fmt.Println(Potatoes(82, 127, 80))
}

File: php/DryingPotatoes/Potatoes.php
<?php

function potatoes($p0, $w0, $p1): int {
    return (int) $w0 * (100 - $p0) / (100 - $p1);
}

print Potatoes(99,100,98); //50
print "\n";
print potatoes(82,127,80); // 114
